### PR TITLE
Uppercase repo_token

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
This pull request aligns the documentation with https://github.com/iterative/cml/pull/489. All the double/single quote changes have been made by the Git hooks, so I guess they're fine. 🙈

### CODE REVIEW

File: content/index.mdx [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-import Header from "components/organisms/SiteHeader"
-import Footer from "components/organisms/SiteFooter"
-import SubscribeSection from "components/molecules/SubscribeSection"
 
-import GitFlowIcon from "media/icons/gitflow.svg"
-import ReportsIcon from "media/icons/reports.svg"
-import BoxIcon from "media/icons/cube.svg"
+import Header from 'components/organisms/SiteHeader'
+import Footer from 'components/organisms/SiteFooter'
+import SubscribeSection from 'components/molecules/SubscribeSection'
```
Comment (casperdcl) [inline content/index.mdx:7]: where do all the changes in this file come from?

Comment (0x2b3bfa0) [reply content/index.mdx:7]: > _All the double/single quote changes have been made by the Git hooks, so I guess they're fine._

I'm baffled by the eccentricities of the commit hooks, but it became like that somehow after running them. 🤔  I wonder if I ran them manually and missed the configuration file.

Comment (casperdcl) [reply content/index.mdx:7]: same with #81 btw

Comment (casperdcl) [reply content/index.mdx:7]: these changed don't look like a formatter/linter lol

**Comment by 0x2b3bfa0** [comment]
@casperdcl, I have manually reverted all the formatter changes, and it doesn't complain.

File: content/index.mdx (hunk 1 above)
Comment (0x2b3bfa0) [reply content/index.mdx]: Please refer to https://github.com/iterative/cml.dev/pull/81#discussion_r690673965

**Review by casperdcl — APPROVED** [review]

**Pull request merged**